docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx

```diff
@@ -164,16 +220,24 @@ Once the WiFi credentials have been successfully transferred to M5 Miner, the de
 ### Type the M5 Miner serial number when prompted
 
 
-<div class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.jpg")} />
-  <span>
+<div class="explanation">
+        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.jpg")} />
```

Comment:
Screw you CSS, I'll do it my way! Does it work ok on mobile view and iPhones?

Reply:
I gave up trying to make M5's look bigger.